Tools/autotest/common.py

```diff
@@ -3527,6 +3577,32 @@ def validator(value2, target2):
 
         self.wait_and_maintain(value_name="Heading", target=heading, current_value_getter=lambda: get_heading_wrapped(timeout), validator=lambda value2, target2: validator(value2, target2), accuracy=accuracy, timeout=timeout, **kwargs)
 
+    def wait_yaw_speed(self, yaw_speed, accuracy=0.2, timeout=30, **kwargs):
+        """Wait for a given yaw speed in degrees."""
+        def get_yawspeed(timeout2):
+            msg = self.mav.recv_match(type='ATTITUDE', blocking=True, timeout=timeout2)
+            if msg:
+                return math.degrees(msg.yawspeed)
+
+        def validator(value2, target2):
+            return math.fabs(value2 - target2) <= accuracy
+
+        self.wait_and_maintain(value_name="YawSpeed", target=yaw_speed, current_value_getter=lambda: get_yawspeed(timeout), validator=lambda value2, target2: validator(value2, target2), accuracy=accuracy, timeout=timeout, **kwargs)
+
+    def wait_speed_vector(self, speed_vector, accuracy=0.2, timeout=30, **kwargs):
+        """Wait for a given speed vector."""
+        def get_speed_vector(timeout2):
+            msg = self.mav.recv_match(type='LOCAL_POSITION_NED', blocking=True, timeout=timeout2)
+            if msg:
+                return Vector3(msg.vx, msg.vy, msg.vz)
+
+        def validator(value2, target2):
+            return math.fabs(value2.x - target2.x) <= accuracy \
```

Comment:
Please use parentheses rather than backslashes.  and the `and` goes on the previous line.